app/vmauth/auth_config.go

```diff
@@ -132,8 +132,8 @@ func (bu *backendURL) isBroken() bool {
 	return ct < atomic.LoadUint64(&bu.brokenDeadline)
 }
 
-func (bu *backendURL) setBroken() {
-	deadline := fasttime.UnixTimestamp() + 3
+func (bu *backendURL) setBroken(dlt time.Duration) {
+	deadline := fasttime.UnixTimestamp() + uint64(dlt.Seconds())
```

Comment:
It seems to me like it makes more sense to use value of `*brokenBackendBackoff` here directly.
Passing it via argument every time makes some overhead while using value directly should work here.

Reply:
fixed it too